src/renderer/families/polkadot/components/CollapsibleList.js

```diff
@@ -61,7 +63,7 @@ const CollapsibleList = ({
   }, []);
 
   return (
-    <Card p={0} mt={24} mb={6}>
+    <Card p={0} mt={24} mb={6} style={containerStyle}>
```

Comment:
why passing a containerStyle (with { margin: 0})  and not removing margins entirely here ?
Or remove the card to only have the collapsible effect ?

I think this component should become generic (and merged with the TokenAccounts show more).

Reply:
yeah sorry i though this component was used elsewhere. indeed a simple box is better. As for a generic way to do this i can do it but i think this one wont benefit from it since it has a very specific way of splitting between visible and hidden items :shrug:

Reply:
@LFBarreto Yup, it is specific, but maybe a good arrangement of collapsible component (only logic), with the atoms & molecules from Design System (table/show more footer...) could allow use to do reusable work.

Not a priority though, there are more important topic. Thanks for the fix.
I have a PR (and some more after) that could lead to conflicts, i'll try to test yours asap - could be great to be merged before we add more conflicts on polkadot.